README.md

```diff
@@ -73,13 +73,22 @@ llama_generator("""q: how are you? a: I am good. How about you? q: What is the w
 
 A detailed example is provided [here](./notebooks/hf_adapted_llama_inference.ipynb).
 
-## Tuning (Coming Soon!!)
+## Tuning
 
-## Training (Coming Soon!!)
+To fine-tune LLaMA, use the `scripts/train_causal.py` training script. Here's
+an example of that command.
+```
+srun --gres=gpu:2 --cpus-per-task=24 --mem=512G --unbuffered \
```

Comment:
do we need to include the srun information here or just torchrun

Reply:
hmmmm.... it felt more complete that way to me since that's how I would always run it, but I suppose it's not required once you're on a host with GPUs.